frontend/src/workspace.rs

```diff
@@ -772,6 +774,7 @@ impl Component for Terminal {
                 onmouseover={self.link.callback(|_| true)}
                 onmouseout={self.link.callback(|_| false)}
             >
+                {format!("{}", &self.props.terminal.label.as_ref().unwrap_or(&"".to_string()))}
```

Comment:
No attention to design of this now. I got lucky that the label appeared inside the terminal square and that I used a single letter label, but more thought needs to happen here.

This would complicate the calculation for how "big" these boxes are for purposes of drawing clear cable lines, but the labels could be outside the box next to terminal, and bumped up above line when line is connected. Or we could widen components with labels to include "sidebar" room to print labels on the inside of box, I suppose.

We could also just show labels on hover, for now.